plugins/generator-1.20.6/neoforge-1.20.6/mappings/screens.yaml

```diff
@@ -109,4 +109,7 @@ Win: WinScreen
 WithNarratorSettings: SimpleOptionsSubScreen
 Working: ProgressScreen
 WorldLoadProgress: LevelLoadingScreen
-WorldSelection: SelectWorldScreen
\ No newline at end of file
+WorldSelection: SelectWorldScreen
+
+#LEGACY
+Realms32bitWarning: WarningScreen
```

Comment:
Since this is a very specific one, I would say you can safely remove it (also from datalist and old mappings)

Reply:
Ok

Reply:
Done